ui/cockpit/client/scripts/directives/change-version.html

```diff
@@ -0,0 +1,25 @@
+<form name="changeVersionForm" ng-if="isActive" class="change-version">
+  <input type="text"
+         class="form-control"
+         id="defVersion"
```

Comment:
Let's not use ids, as we can have multiple instances of this widget (migration view)